pkg/controller/tas/resource_flavor.go

```diff
@@ -158,13 +161,16 @@ func (h *topologyHandler) Create(ctx context.Context, e event.CreateEvent, q wor
 		return
 	}
 
+	defer h.notifyTopologyWatchers(nil, topology)
+
 	// trigger reconcile for TAS flavors affected by the node being created or updated
```

Comment:
Is the reconcile loop triggered if the nodes are created/updated or topology?

Reply:
topology, that was a stale comment. Fixed.